lib/rubycritic/core/location.rb

```diff
@@ -18,6 +18,10 @@ def to_s
       "#{pathname}:#{line}"
     end
 
+    def to_s_with_realpath
```

Comment:
Since this logic only exists for the emacs renderer it could be just a private method on the `RenderTool` class - what do you think?